### ISSUE TITLE
Example of how to use custom config object?

### ISSUE THREAD

**Issue by RCheesley** [thomasschwarz96/grav-plugin-simple-cookie#6]
I was looking for an example of how to use the custom config object - e.g. to load google analytics only when the user accepts the cookie warning. Do you have an example you could share of how this could be achieved? Thanks!

**Comment by qw3rtty** [comment]
@RCheesley 
The correct format for the "Custom config" field is an JavaScript-Object which can be encode/decode to an JSON string.

Here is a working example:
`{"revokable":false, "revokeBtn": "<a class='test'>REVOKE</a>"}`

Here is a link to the documentation of cookie consenst for all available options:
https://www.osano.com/cookieconsent/documentation/javascript-api/

Right now, it is not possible to configure a callback function for "onStatusChange" for example.
So right now you can't load an cookie when a user accepts the cookies.
The reason is, JSON.stringify or JSON.parse can't work with functions.

I will solve this problem and let you know when i got a fix for it.

**Comment by Mehditerranee** [comment]
@tomschwarz Is there any news about this point ? 

FYI, i have tested the example you provided and it doesn't work : i got error 500.

Could you provide a complete example of a simple-cookie.yaml file  with a custom configuration (custom-config) please ?

**Comment by Mehditerranee** [comment]
@RCheesley Did you manage to use the custom-config parameter ? If yes, could you explain me how please ?

**Comment by RCheesley** [comment]
@Mehditeranee no we did not - unfortunately meaning we are unable to fully comply with GDPR requirements.

**Comment by Mehditerranee** [comment]
@RCheesley Ok thank you for your feedback. I don't understand how a CMS like Grav doesn't provide a full and strong solution for RGPD compliance. (because, in this case, the plugin documentation is poor and seems to not manage advanced use cases)

**Comment by hughbris** [comment]
I agree with the sentiment here, this feature is no good until documented properly.

> I don't understand how a CMS like Grav doesn't provide a full and strong solution for RGPD compliance.

@Mehditerranee Grav doesn't have this because it's a lightweight multi-purpose CMS, not everyone needs this, and it's better achieved with plugins. Also, this allows you to take your pick of them.
